django_emarsys/util.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from django.apps import apps
+from django.conf import settings
+
+log = logging.getLogger(__file__)
+
+
+def get_model(model_string):
+    chunks = model_string.rsplit('.', 1)
+    if len(chunks) == 2:
+        app_label, model_name = chunks
+    else:
+        app_label = None
+        model_name = model_string
+
+    return apps.get_model(app_label=app_label, model_name=model_name)
```

Comment:
Nice, I should read more than the signature sometimes.

It results in an error. The question is how to handle that case in that function. It cannot recuperate anyway.
Using the `apps.get_model(model_string)` directly will be easier, but it throws `ValueError("need more than 1 value to unpack")`. Oh well.